ideascube/mediacenter/tests/factories.py

```diff
@@ -4,12 +4,15 @@
 
 from ..models import Document
 
+class EmptyFileField(factory.django.FileField):
+    DEFAULT_FILENAME = None
 
 class DocumentFactory(factory.django.DjangoModelFactory):
     title = factory.Sequence(lambda n: "Test document {0}".format(n))
     summary = "This is a test summary"
     lang = settings.LANGUAGE_CODE
     original = factory.django.FileField()
+    preview = EmptyFileField()
```

Comment:
Seems the only alternative would have been to change all the existing tests so that they'd pass `preview=None` when instanciating the factory.

meh...